src/solver/simulation/common-eco-adq.cpp

```diff
@@ -113,65 +113,6 @@ static void RecalculDesEchangesMoyens(Data::Study& study,
     }
 }
 
-void PrepareDataFromClustersInMustrunMode(Data::Study& study,
```

Comment:
This function (PrepareDataFromClustersInMustrunMode) has many drawbacks : 
- It's too long
- It's supposed to offer a treatment common to mode **adequacy** and **economy** but its body clearly separates **adequacy** mode and the other modes (**economy**)
- it's complicated : for example, we have several loop over thermal clusters of an area though we could only have one.

So we removed this function from **common-eco-adq.cpp** and put down these calculations in specialized function related to each simulation mode.